quadrat/sass/blocks/_post-template.scss

```diff
@@ -0,0 +1,3 @@
+.wp-block-post-template {
+    margin: 18px 0;
```

Comment:
I've added vertical margin to this `ul` as before we were reliant on the browser defaults. This means the spacing should be consistent between browsers and matches the other spacing changes for keeping the post meta in the same position between templates.